Index: OpenCC-ver.1.0.5/data/CMakeLists.txt
===================================================================
--- OpenCC-ver.1.0.5/data/CMakeLists.txt
+++ OpenCC-ver.1.0.5/data/CMakeLists.txt	2018-04-13 10:34:55.500627251 +0200
@@ -112,7 +112,8 @@
     COMMAND
       ${CMAKE_COMMAND} -E copy "$<TARGET_FILE:libopencc>" "$<TARGET_FILE_DIR:${OPENCC_DICT_BIN}>"
     COMMAND
-      ${OPENCC_DICT_BIN}
+      LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/src:ENV{LD_LIBRARY_PATH}
+      ${CMAKE_BINARY_DIR}/src/tools/${OPENCC_DICT_BIN}
         --input ${DICT_${DICT}_INPUT}
         --output ${DICT}.ocd
         --from text
