--- CMakeLists.txt.lib64	2010-12-23 14:18:23.000000000 +0100
+++ CMakeLists.txt	2011-01-29 20:03:08.000000000 +0100
@@ -55,8 +55,8 @@
 ######## Directory
 
 set (DIR_PREFIX ${CMAKE_INSTALL_PREFIX})
-set (DIR_LIBRARY ${CMAKE_SHARED_LIBRARY_PREFIX})
-set (DIR_LIBRARY_STATIC ${CMAKE_STATIC_LIBRARY_PREFIX})
+set (DIR_LIBRARY ${CMAKE_INSTALL_LIBDIR})
+set (DIR_LIBRARY_STATIC ${CMAKE_INSTALL_LIBDIR})
 set (DIR_INCLUDE include)
 set (DIR_SHARE share)
 set (DIR_BIN bin)
@@ -88,4 +88,4 @@
 	add_subdirectory(po)
 endif (GETTEXT_FOUND AND ENABLE_GETTEXT)
 
-add_subdirectory(test)
\ No newline at end of file
+add_subdirectory(test)
